Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -920,17 +1080,17 @@ public void TestBocListWithNoFakeTableHeader ()
       var bocList = home.Lists().GetByLocalID("JobList_NoFakeTableHeader");
 
       bocList.ClickOnSortColumn("EndDate");
-      Assert.That(bocList.GetRow(1).GetCell(1).GetText(), Is.EqualTo("CEO"));
+      Assert.That(bocList.GetRow(1).GetCell(2).GetText(), Is.EqualTo("CEO"));
 
       bocList.ClickOnSortColumn("EndDate");
       bocList.ClickOnSortColumn("EndDate");
-      Assert.That(bocList.GetRow(1).GetCell(1).GetText(), Is.EqualTo("Programmer"));
+      Assert.That(bocList.GetRow(1).GetCell(2).GetText(), Is.EqualTo("Programmer"));
 
       var row = bocList.GetRowWhere("Title", "Developer");
       Assert.That(row.GetCell("DisplayName").GetText(), Is.EqualTo("Developer"));
 
       var columnTitles = bocList.GetColumnDefinitions().Select(cd => cd.Title);
-      Assert.That(columnTitles, Is.EquivalentTo(new[] { "Title", "StartDate", "EndDate", "DisplayName" }));
+      Assert.That(columnTitles, Is.EquivalentTo(new[] { "", "Title", "StartDate", "EndDate", "DisplayName" }));
```

Comment:
Do you think it would help the test readability if we added a column title to the edit-column (e.g. "Edit")?